src/ServiceControl/Bootstrapper.cs

```diff
@@ -56,7 +50,7 @@ public Bootstrapper(Settings settings, BusConfiguration configuration)
         private void Initialize()
         {
             var loggingSettings = new LoggingSettings(settings.ServiceName);
-            ConfigureLogging(loggingSettings);
+            RecordStartup(loggingSettings);
```

Comment:
I made it so the `LoggingSettings` ctor does not log anything. The initialization of the logging framework now happens in the entry point for the console application. This is actually the 2nd time we create a `LoggingSettings` instance in the lifecycle of the application. It was easier to create a new instance than to pass the instance from the entry point all the way to the bootstrapper.